main.go

```diff
@@ -305,6 +321,27 @@ func healthCheck(cfg config.AppConfig, logger log.Logger) error {
 	return nil
 }
 
+func printVersion(writer io.Writer) error {
+	var buffer bytes.Buffer
+	var libcontainersshVersion string
+	bi, ok := debug.ReadBuildInfo()
+	if !ok {
+		return fmt.Errorf("read build info %t", ok)
+	}
+	for _, dep := range bi.Deps {
+		if dep.Path == "go.containerssh.io/libcontainerssh" {
+			libcontainersshVersion = dep.Version
+		}
+	}
+	buffer.WriteString("libcontainerssh version ")
+	buffer.WriteString(libcontainersshVersion)
+	buffer.WriteString("\n")
+	if _, err := writer.Write(buffer.Bytes()); err != nil {
+		return fmt.Errorf("failed to write Version information (%w)", err)
+	}
```

Comment:
That's good I think it's also a good idea to add vcs.{revision,modified,time} from [here](https://pkg.go.dev/runtime/debug#ReadBuildInfo) that'll say the version of ContainerSSH/the repo it was built from.

Perhaps a format it the form of 
```
libcontainerssh: v0.5
build revision: v0.5(dirty) {{build time}}
```
You should check for the existence of these tags individually as not all might exist e.g. build time might be omitted for reproducibility.